src/docs/ftw-introduction/01-how-to-customize-ftw/index.md

```diff
@@ -35,11 +35,10 @@ couple of articles first:
 
 - [Introducing Flex](/introduction/introducing-flex/)
 - [What development skills are needed?](/introduction/development-skills/)
-- [Getting started](/tutorial/introduction/)
+- [Getting started](/introduction/getting-started-with-ftw-daily/)
 
-In the last link (Getting started), you can see a rough guide how to
-setup a Github account:
-[Prerequisites for tutorial](http://localhost:8000/tutorial/introduction/#prerequisites).
+The [Tutorial introduction](/tutorial/introduction/#prerequisites) will
+also walk you through creating a Github repository.
```

Comment:
I'm not sure how strict we are on other pages in Docs, but they still write their name as *GitHub* with a capital H, so maybe that could be followed here, too ;-)

Reply:
Good point! I'll go ahead and make this change directly through the GitHub markdown editor if that is fine @Gnito ? Or should I create a pull request?

Reply:
markdown editor is fine, but if you have this repo locally, you can also just create new commit and push the update here before merging. (With FTW templates, the latter would be the way to go -  but since this is our own docs site, it's a bit more relaxed what kind of commits we make.)

Reply:
Ok, good to know. I have now created a new commit and pushed the update to this branch. Go ahead and merge if everything looks fine.